### PR TITLE
Support npm specifiers

### PR DESCRIPTION
**Pull request by WilliamBergamin**
### Type of change (place an x in the [ ] that applies)

- [ ] New sample
- [ ] New feature
- [x] Bug fix
- [ ] Documentation

### Summary

This PR aims to update the version of the `deno-slack-hooks` package and enable the project to use `npm` specifiers to import packages

This also resolves #32 

### Testing

1. Pull this branch
2. Run `slack deploy`
3. The project should bundle and deploy as expected

### Requirements

- [x] I’ve checked my submission against the Samples Checklist to ensure it complies with all standards
- [x] I have ensured the changes I am contributing align with existing patterns and have tested and linted my code
- [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct)

### CODE REVIEW

**Comment by filmaj** [comment]
Should this be created as a `pre-release` branch for the upcoming release? Since it includes a freshly-cut SDK version bump, and the automated tests will rely on the existence of a `pre-release` branch?

**Comment by WilliamBergamin** [comment]
I think this PR or the `pre-release` PR will update the package correctly, just note that the npm specifiers will need to be updated in the samples

File: functions/open_form/blocks.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import {
   MultiUsersSelect,
   Option,
   SectionBlock,
-} from "https://cdn.skypack.dev/@slack/types?dts";
+} from "npm:@slack/types";
```
Comment (filmaj) [inline functions/open_form/blocks.ts:11]: I'd version this, the current version is 2.9.0. I am planning on releasing a 3.0 version soon-ish which will include breaking changes.

**Review by filmaj — CHANGES_REQUESTED** [review]
Should probably use a versioned `types` package and also move it into a `deps.ts` if it exists?

This PR will also conflict with #34 FYI.

**Review by Nidacoke — APPROVED** [review]

**Review by filmaj — APPROVED** [review]
LGTM but def should be tested with an actual `slack deploy` / at least a deno-hooks `build` hook invocation to see if esbuild can handle it or not.

Would be nice to add that to samples' CI, too: can the project be built using the deno `build` hook?

**Comment by WilliamBergamin** [comment]
🟢 tested with `slack deploy` everything works as expected
🟢 tested with `deno run ... https://deno.land/x/deno_slack_hooks@1.3.0/build.ts` everything builds as expected

**Pull request merged**